airflow/providers/microsoft/azure/CHANGELOG.rst

```diff
@@ -27,6 +27,20 @@
 Changelog
 ---------
 
+8.0.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+.. warning::
+   In this version of the provider, we have removed network_profile param from AzureContainerInstancesOperator
+
+* Remove ``network_profile`` param from ``AzureContainerInstancesOperator``
+* Remove deprecated ``extra__azure__tenantId`` from azure_container_instance connection extras
+* Remove deprecated ``extra__azure__subscriptionId`` from azure_container_instance connection extras
```

Comment:
Should we mention the named tuple `Volume` has been removed as well?

Reply:
I have reverted this PTAL